web/styles/settings.css

```diff
@@ -1786,7 +1788,25 @@ label.preferences-radio-choice-label {
 
     #settings_page {
         .settings-header.mobile {
-            display: block;
+            display: flex;
+            justify-content: space-between;
+            height: 45px;
+
+            .fa-chevron-left {
+                position: static;
```

Comment:
Can we remove the relative positionning set for this, since `display` is set to `none` for larger widths and for smaller widths the relative positioning properties are overridden here.

Reply:
Yes! Good call. Adding this diff to the main commit:

```diff
diff --git a/web/styles/settings.css b/web/styles/settings.css
index b3656f5ec9..53f749c751 100644
--- a/web/styles/settings.css
+++ b/web/styles/settings.css
@@ -1444,9 +1444,6 @@ label.preferences-radio-choice-label {
 
             .fa-chevron-left {
                 float: left;
-                position: relative;
-                top: 14px;
-                left: 10px;
             }
         }
```

Reply:
Done, thanks!